### ISSUE TITLE
Feature Requests: Please allow notify

### ISSUE THREAD

**Issue by hvorragend** [jcwillox/home-assistant-blueprints#1]
Could you please allow to configure one of the several notifier for push notifications?
 
I am not using the HA-App. 

Thank you.

**Comment by jcwillox** [comment]
Fair point that would be good to have, I based this off the update notifications blueprint so I just included what they had. Supporting any notifier is a little more complicated, with the mobile app it's easy just select the mobile devices and they're all notified in the same way.

The easiest solution I can think of may just be to have a "Notify Action" option which would allow you to configure a service to be called and provide template variables to set the message and title.
